_includes/releases/v22.1/v22.1.12.md

```diff
@@ -0,0 +1,104 @@
+- When configured to true, the `sql.metrics.statement_details.dump_to_logs` cluster setting no longer causes a mutex deadlock. [#92278][#92278]
+- Fixdc incorrect cancellation logic when attempting to detect stuck range feeds. [#92702][#92702]
+- Fixed an internal error when comparing a tuple type with a non-tuple type. [#92714][#92714]
+- `attidentity` for `GENERATED BY DEFAULT AS IDENTITY` column should be `d`.  <what was there before: Previously, ...> <why it needed to change: This was inadequate because ...> <what you did about it: To address this, this patch ...> [#92835][#92835]
+- CockroachDB previously could incorrectly evaluate queries that performed left semi and left anti "virtual lookup" joins on tables in `pg_catalog` or `information_schema`. These join types can be planned when a subquery is used inside of a filter condition. The bug was introduced in 22.1.0 and is now fixed. [#92881][#92881]
+
+<h3 id="v22-1-12-performance-improvements">Performance improvements</h3>
+
+- GC score triggering mvcc GC run is lowered. GC Score is ratio is computed from mvcc stats and uses ratio of live objects and estimated garbage age to estimate collectability of old data. Reduced score will trigger GC earlier lowering interval between runs 3 times, giving 2 times reduced peak garbage usage at the expense of 30% increase of wasteful data scanning on constantly updated data. [#92816][#92816]
+- CockroachDB in some cases now correctly incorporates the value of the `OFFSET` clause when determining the number of rows that need to be read when the `LIMIT` clause is also present. Note that there was no correctness issue here - only that extra unnecessary rows could be read. [#92839][#92839]
+- `SHOW BACKUP` on a backup containing several table descriptors is now more performant. Release justification: low risk performance improvement required for the use of schedules in CockroachCloud [#93143][#93143]
+
+<h4 id="v22-1-12-changes-without-release-note-annotation">Changes without release note annotation</h4>
+
+- [#91703][#91703] [949e22e5c][949e22e5c] ui: update cluster ui to v22.1.10 (Marylia Gutierrez <marylia@cockroachlabs.com>)
+- [#91485][#91485] [ff54be2a7][ff54be2a7]  (Chengxiong Ruan <chengxiongruan@gmail.com>, Steven Danna <danna@cockroachlabs.com>)
+
+<h3 id="v22-1-12-contributors">Contributors</h3>
+
+This release includes 75 merged PRs by 37 authors.
+We would like to thank the following contributors from the CockroachDB community:
+
+- Mark Sirek (first-time contributor, CockroachDB team member)
```

Comment:
I think you want to delete the first two contributors here as they are team members.